src/bptl/work_units/zac/models.py

```diff
@@ -0,0 +1,42 @@
+from typing import Dict, Union
+
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from solo.models import SingletonModel
+
+
+class ZACConfig(SingletonModel):
+    api_root = models.URLField(
+        _("API-root"),
+        help_text=_("Root URL of ZAC api"),
+        default="https://zac.example.com",
+    )
+
+    header_key = models.CharField(
+        _("header key"),
+        max_length=100,
+        blank=True,
+        help_text=_("HTTP Authorization header name, required if the API is not open."),
+    )
+    header_value = models.CharField(
+        _("header value"),
+        max_length=255,
+        blank=True,
+        help_text=_(
+            "HTTP Authorization header value, required if the API is not open."
+        ),
+    )
```

Comment:
Hrm, I'd rather see a FK to `zgw_consumers.Service` for this instead, no? You already configure the API root in that model + the auth headers. OTOH that's also how BRP is done now, so probably we should just clean it up all in one go before 1.0. I've created #89 for that.

Reply:
Okay. Leaving it in for now then.